README.md

```diff
@@ -50,7 +50,10 @@ conventions.
 ## Editing the manual using git (recommended)
 
 * Clone the repository `git clone https://github.com/mixxxdj/manual.git`
+* Install requirements `pip install -r requirements.txt` (see above)
+* Install pre-commit `pre-commit install`
 * Perform changes
+* Check changes `pre-commit run --files /path/to/your/file`
```

Comment:
This is unneeded, it will automatically check all changed files on `git commit`.

Reply:
yes it is needed because the pre-commit will only complain sth is wrong when you push a commit but will not fix or prevent pushing. so you gonna get commits that have whatever issues still in there, checks go wrong here on Git. Instead the user should only commit things if locally the pre-commit is passed. Another positive effect: local executed pre-commit will modify the file (eg deleting whitespaces).

Reply:
> yes it is needed because the pre-commit will only complain sth is wrong when you push a commit but will not fix or prevent pushing.

No. When you run `git commit` it checks all files in that commit automatically and rejects the commit when something is wrong. Running `pre-commit run --files`/`pre-commit run --all-files` is only necessary if you start to use pre-commit *after* you already committed/pushed changes with issues.

Btw, If you also want to prevent pushes, you can run `pre-commit install -t pre-push`.

Reply:
Either remove this line or replace it with:

```suggestion
* Build the HTML manual locally using `sphinx-build -W -b html source build/html` and check the output in the `build/html` directory in your browser
```

Reply:
... that is also good. Why not just `make html` ?

I got a warning using `sphinx-build -W -b html source build/html`
```
Warning, treated as error:
dot command 'dot' cannot be run (needed for graphviz output), check the graphviz_dot setting
```

Reply:
> ... that is also good. Why not just `make html` ?

`make html` requires make to be installed. I don't think that's the case on most windows systems. It also doesn't specify `-W` (to break on warnings) and IMHO is just useless indirection. :smile: 

>     Warning, treated as error:
>     dot command 'dot' cannot be run (needed for graphviz output), check the graphviz_dot setting

You need to install graphviz (e.g. via your package manager or by [downloading it on this website](https://www.graphviz.org/download/) if youre on windows).

Our README is wrong in that regard: Pip only installs the python bindings, it doesn't replace the actual graphviz installation. Can you correct that too while you're at it?